DELIVERY.README.txt

```diff
@@ -1,124 +1,127 @@
---------------------------------------------------------------------
-Contains the final vcf files, in total 3 vcf files.
-
-—-SampleN.clean.dedup.recal.bam.genomic.vcf
-gVCF was developed to store sequencing information for both variant 
-and non-variant positions
-
---SampleN.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf
-vcf file containing indel variants called from the final bam file and annotated with 
-variation effects generated with snpEff.
-
---SampleN.clean.dedup.recal.bam.recalibrated.snp.annotated.vcf
-vcf file containing snp variants called from the final bam file and annotated with 
-variation effects generated with snpEff.
+README
+=======
+
+The README describes the content of the delivered data.
+-----------------------------------------------------------------
+
+Root level
+----------
+The root folder, which is named by the project id, contains one report folder
+and one folder for each sample.
+
+|--ProjectID
+   |--00-Report
+   |--Sample1
+   |--Sample2
+   ...
+   ...
+   |--SampleN
+
+====================================================================
+ProjectID -> 00-Report
+====================================================================
+00-Report folder contain sequence QC data, which provide
+information about the quality and other features of the fastq-files, both on
+sample and lane level. The reports are organized by sequence run.
+
+|--ProjectID
+   |--00-Report
+      |--Runfoldername1
+      |--Runfoldername2
+      ...
+      ...
+      |--RunfoldernameN
+
+====================================================================
+ProjectID -> SampleN
+====================================================================
+Each sample will contain the following subfolders:
+
+|--ProjectID
+   |--SampleN
+      |--00-Reports
+      |--01-QC
+      |--02-FASTQ
+      |--03-BAM
+      |--04-VCF
+
+ProjectID -> SampleN -> 00-Reports
+--------------------------------------------------------------------
+Contains three type of reports, version_report.txt, snpEff summary
+and sample report.
+
+--version_report.txt
+Information from piper, about data sources and software version
+that have been used.
+
+--snpEff summary report
+SnpEff generated report.
+
+--Sample report
+Report summarizing information about the analysis, alignment and variants.
+
+ProjectID -> SampleN -> 01-QC
+--------------------------------------------------------------------
+--SampleName.clean.dedup.recal.qc
+Qualimap QC report from mapping
+
+--SampleName.metrics
+Picard Mark duplicates output
+
+--SampleName.clean.dedup.recal.bam.snp.eval
+GATK Variant calling evaluation for snp
+
+--SampleName.clean.dedup.recal.bam.indel.eval
+GATK Variant calling evaluation for indels
+
+ProjectID -> SampleN -> 02-FASTQ
+--------------------------------------------------------------------
+--bam2fastq.sh
+SLURM (or bash) script that can be used for generating FASTQ files from
+a BAM file. The script is formatted to be submitted and run on a SLURM
```

Comment:
maybe here is more fair to say "to be submitted and run on Uppmax milou cluster." due to the module load